src/CarbonAware.WebApi/src/Program.cs

```diff
@@ -1,11 +1,12 @@
 using CarbonAware;
-using CarbonAware.Aggregators.Configuration;
-using CarbonAware.WebApi.Configuration;
 using CarbonAware.WebApi.Filters;
+using GSF.CarbonAware.Configuration;
+using GSF.CarbonAware.Exceptions;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using CarbonAware.WebApi.Configuration;
```

Comment:
Can you alphabetize these?

Reply:
Sorted usings for this file and a few others 👍